docs/building-a-form/available-components.md

```diff
@@ -338,7 +376,37 @@ Use this feature to collect multiple items with the same form questions, such as
 
 #### Usage guidelines
 
-In `formConfig`, define this in the data definition.
+In order to display multiple items with the same form questions, the data in the `schema` must be defined as `type: 'array'`, with each group of questions an `item` in that `array`. The `schema` and `uiSchema` for the group of questions within the `items` object is structured the same as other fields.
```

Comment:
> In order to display multiple items with the same form questions, the data in the `schema` must be defined as `type: 'array'`, with each group of questions an `item` in that `array`.

A little tighter:

"To display multiple groups of the same form questions, define the data in the `schema` as `type: 'array'`, with each group of questions an `item` in that `array`."